SocialCareCaseViewerApi/V1/Boundary/Requests/CreateReferralRequest.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using FluentValidation;
+
```

Comment:
I think we should look to utilise #nullable enable here

We can then also drop the [Required] FluentValidation attributes

Reply:
Done in 2f5de34